package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "private": true,
+  "name": "dolt-sql-workbench",
+  "productName": "dolt-sql-workbench",
+  "version": "1.0.0",
+  "main": "app/background.js",
+  "workspaces": [
+    "graphql-server",
+    "web"
+  ],
+  "packageManager": "yarn@4.0.2",
+  "scripts": {
+    "dev": "nextron",
+    "build": "nextron build",
+    "dist-all": "electron-builder -mwl",
+    "postinstall": "electron-builder install-app-deps",
+    "clean": "tsc -b --clean && yarn yalc:clean",
+    "compile": "tsc -b"
```

Comment:
Should these use the `clean` and `compile` scripts from the workspaces instead? Like we do in `ld/web`